common/src/main/java/com/wynntils/mc/render/FontRenderer.java

```diff
@@ -213,11 +219,14 @@ private float calculateRenderHeight(List<TextRenderTask> toRender) {
                             .size();
                 }
 
-                height += lines * font.lineHeight + NEWLINE_OFFSET * (lines - 1);
+                height += lines * font.lineHeight;
+                totalLineCount++;
```

Comment:
@kristofbolyai Should that not be like `totalLineCount += lines`?

Reply:
At one point I thought so, I will check.

Reply:
Tested this, seems to work fine.

Reply:
I'm just confused as to how this can be correct. Can you explain, apart from just "it seems to work"? If not, maybe the logic needs some more straightening out so that it becomes apparent.

Reply:
I will take a look.